src/Cthulhu.jl

```diff
@@ -189,23 +189,25 @@ function lookup(interp::CthulhuInterpreter, mi::MethodInstance, optimize::Bool)
     (codeinf, rt, infos, slottypes::Vector{Any})
 end
 
+const debuginfo_keys = [:none, :compact, :source]
+
 ##
 # _descend is the main driver function.
 # src/reflection.jl has the tools to discover methods
 # src/ui.jl provides the user facing interface to which _descend responds
 ##
 function _descend(interp::CthulhuInterpreter, mi::MethodInstance; override::Union{Nothing, InferenceResult} = nothing, iswarn::Bool, params=current_params(), optimize::Bool=true, interruptexc::Bool=true, verbose=true, kwargs...)
-    debuginfo = true
+    debuginfo = 1 # default is compact debuginfo
```

Comment:
Any specific reason to not use 1-based index here ?

Reply:
Not really. Using 0 for no debuginfo felt a little more natural, but I can also just change it to use one based indexing.

Reply:
imho something like `debuginfo_keys[debuginfo + 1]` looks a bit verbose, but I'm okay with either way. Your choice :)

Reply:
The right thing to do is to use enums ;)

Reply:
Does this look like a reasonable compromise?